COMRS-66: Internationalization of Location Button

prashantsable1991 opened the pull request:
Internationalization of Location Button
JIRA Card: https://jembiprojects.jira.com/browse/COMRS-66

Commit:
COMRS-60 Add cameroon locations

prashantsable1991 committed:
COMRS-66 Internationalization of Location Button

lauravignoli committed:
Revert "COMRS-60 Add cameroon locations"

This reverts commit 379008c3ad7280c1220fcd259463285beab8b940.

tmvumbi2 commented on configuration/locations/locations.csv:
```diff
@@ -0,0 +1,31 @@
+Uuid,Void/Retire,Name,Description,Parent,Tags,Address 1,Address 2,Address 3,Address 4,Address 5,Address 6,City/Village,County/District,State/Province,Postal Code,Country,_order:1000
```
@prashantsable1991 there are some small differences between the original location hierarchy structure and the imported structure:
- HOME_LOCATION_LABOR_WARD should be under HOME_LOCATION_MATERNITY
- HOME_LOCATION_PEDIATRIC should be under HOME_LOCATION_SPECIALIZED_CONSULTATIONS
- Subcenter 1, 2 and 3 should be voided/retired
- The tree starting from 'Ganiyari' should to be removed (or voided if it can't be removed through configuration)
- Unknown Location should be removed (or voided if it can't be removed through configuration)

Could you also please double check in case I forgot something?

Original location hierarchy:
![image](https://user-images.githubusercontent.com/22587098/53242856-b1666d00-36ae-11e9-9d72-668aea7bf203.png)


Imported location hierarchy:
![image](https://user-images.githubusercontent.com/22587098/53242871-bfb48900-36ae-11e9-935a-f2c3e60cfc38.png)

prashantsable1991 replied:
Pushed updated location.csv

tmvumbi2 reviewed (changes requested):
Thanks @prashantsable1991, the location import worked well. I left a feedback regarding some small differences between the original location hierarchy and the import result

prashantsable1991 committed:
COMRS-66 | Internationalization of Location Button

prashantsable1991 committed:
COMRS-66 | Internationalization of Location Button

prashantsable1991 committed:
Merge branch 'COMRS-66' of https://github.com/jembi/cameroon-bahmni-metadata into COMRS-66

prashantsable1991 committed:
COMRS-66 | Internationalization of Location Button

tmvumbi2 committed:
COMRS-66 Remove 'LOCATION_1' from location list

LOCATION_1 was committed by error (it was used for testing)

tmvumbi2 committed:
COMRS-66 Void the locations not used for Cameroon

This commit retires the default locations coming with Bahmni that will not be used in the Cameroon implementation
